libs/react-helpers/src/lib/format/number.ts

```diff
@@ -4,7 +4,7 @@ import memoize from 'lodash/memoize';
 import { getUserLocale } from './utils';
 
 export function toDecimal(numberOfDecimals: number) {
-  return Math.pow(10, -numberOfDecimals);
```

Comment:
this never worked. because javascript is javascript 
<img width="307" alt="Screenshot 2022-08-16 at 18 36 02" src="https://user-images.githubusercontent.com/16125548/184949792-7794c251-04a4-4949-9eed-d9c723d894e9.png">